tests/integration/fuzz/MerkleVCA.t.sol

```diff
@@ -34,7 +34,18 @@ contract MerkleVCA_Fuzz_Test is Shared_Fuzz_Test {
     /// the instance of `merkleVCA` deployed in {Integration_Test}.
     function testFuzz_ClaimAndForgoneAmount(uint128 fullAmount, uint40 claimTime) external {
         uint128 actualClaimAmount = merkleVCA.calculateClaimAmount(fullAmount, claimTime);
-        uint128 actualForgoneAmount = merkleVCA.calculateForgoneAmount(fullAmount, claimTime);
+        uint128 actualForgoneAmount;
+        if (claimTime > 0 && claimTime < VCA_START_TIME) {
```

Comment:
I suggest we follow the best practices and put it in a different test: `testFuzz_RevertWhen_ClaimTimeLessThanStartTime`